service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -2612,16 +2572,6 @@ protected FileIO loadFileIO(String ioImpl, Map<String, String> properties) {
         callContext, ioImpl, properties, identifier, locations, storageActions, resolvedPath);
   }
 
-  private void blockedUserSpecifiedWriteLocation(Map<String, String> properties) {
```

Comment:
`loadFileIO` is also unused and should be removed as well.

Reply:
Agreed that it can be removed, but unlike `blockedUserSpecifiedWriteLocation` it's not related to this PR so I think we should do that separately.